rpcs3/Emu/Cell/lv2/sys_usbd.h

```diff
@@ -1,18 +1,54 @@
 #pragma once
 
+namespace vm { using namespace ps3; }
+
 // SysCalls
 
-s32 sys_usbd_initialize();
+struct DeviceListUnknownDataType
+{
+	u8 unk1;
+	u8 deviceID;
+	u8 unk3;
+	u8 unk4;
+} ;
```

Comment:
weird space.

and the tab-spacings in the next two structs seem off

xD

Reply:
ah, good ol' copy-pasta.

Addressed.